tox.ini

```diff
@@ -1,8 +1,7 @@
 [tox]
-envlist = py3, pep8
+envlist = py3, py35, py36, pep8
 
```

Comment:
Please see my previous comments, you have to define basepython=python3 for py3 and pep8 test envs.